internal/v2/s2av2.go

```diff
@@ -84,18 +88,22 @@ func NewServerCreds(s2av2Address string, localIdentities []*commonpbv1.Identity,
 	// Create an AccessTokenManager instance to use to authenticate to S2Av2.
 	accessTokenManager, err := tokenmanager.NewSingleTokenAccessTokenManager()
 	if err != nil {
-		return nil, err
+		accessTokenManager = nil
```

Comment:
Same here, should we just delete this if statement?

Reply:
Good point! Done.